test/perf.js

```diff
@@ -112,7 +114,7 @@ test("validateSingle", (t) => {
         for (i = 0; i < ITERATIONS; i++) {
           // validate a single message (`seq` == 1)
           const start = Date.now();
-          validate.validateSingle(msgs[0], null, () => {
+          validate.validateSingle(msgs[0].value, null, () => {
```

Comment:
Perhaps we could rename `msgs` to `kvtMsgs` to preserve consistency?

Reply:
Yes, good call 👍🏻